assets/app/scripts/services/data.js

```diff
@@ -605,15 +613,21 @@ angular.module('openshiftConsole')
       protocol = window.location.protocol === "http:" ? "http" : "https"; 
     }
 
+    var namespaceInPath = params.namespace && SERVER_TYPE_MAP[type].namespacePath;
+    var namespace = null;
+    if (namespaceInPath) {
+      namespace = params.namespace;
+      params.namespace = null;
```

Comment:
Do we need to worry about mutating params? Not sure what calls this and if it expects namespace to be unset on the params object it passes in